README.md

```diff
@@ -38,91 +36,147 @@
+- Contact us:
+  - [Ask questions and discuss problems on the StellarGraph Discourse forum](https://community.stellargraph.io)
+  - [File an issue](https://github.com/stellargraph/stellargraph/issues/new/choose)
+
+
+## Example: GCN
+
+One of the earliest deep machine learning algorithms for graphs is a Graph Convolution Network (GCN) [6]. The following example uses it for node classification: predicting the class from which a node comes. It shows how easy it is to apply using StellarGraph, and shows how StellarGraph integrates smoothly with Pandas and TensorFlow and libraries built on them.
+
+#### Data preparation
+
+Data for StellarGraph can be prepared using common libraries like Pandas and scikit-learn.
+
+``` python
+import pandas as pd
+from sklearn import model_selection
+
+# load data into Pandas DataFrames, e.g. from CSV files or a database
+nodes, edges, targets = load_my_data()
+
+# Use scikit-learn to compute training and test sets
+train_targets, test_targets = model_selection.train_test_split(targets, train_size=0.5)
+
+```
+#### Graph machine learning model
 
-* Recommender systems
-  - See the demo in folder `demos/link-prediction-hinsage` for an example of how to predict
-  movie ratings between users and movies using a Heterogeneous generalisation of GraphSAGE model, which we call HinSAGE.
+This is the only part that is specific to StellarGraph. The machine learning model consists of some graph convolution layers followed by a layer to compute the actual predictions as a TensorFlow tensor. StellarGraph makes it easy to construct all of these layers via the `GCN` model class. It also makes it easy to get input data in the right format via the `StellarGraph` graph data type and a data generator.
 
--->
+```python
+import stellargraph as sg
 
+# convert the raw data into StellarGraph's graph format for faster operations
+graph = sg.StellarGraph(nodes, edges)
+
+generator = sg.mapper.FullBatchNodeGenerator(graph, method="gcn")
+
+# two layers of GCN, each with hidden dimension 16
+gcn = sg.layer.GCN(layer_sizes=[16, 16], generator=generator)
+x_inp, x_out = gcn.build() # create the input and output TensorFlow tensors
+
+# use TensorFlow Keras to add a layer to compute the (one-hot) predictions
+predictions = tf.keras.layers.Dense(units=len(ground_truth_targets.columns), activation="softmax")(x_out)
+```
+
+#### Training and evaluation
+
+These input and output tensors can be used to create a TensorFlow Keras model and work with it such as training and evaluation with the functions offered by Keras. StellarGraph's data generators make it simple to construct the required Keras Sequences for input data.
+
+```python
+import tensorflow as tf
+
+model = tf.keras.Model(inputs=x_inp, outputs=predictions)
+model.compile("adam", loss="categorical_crossentropy", metrics=["accuracy"])
+
+model.fit(generator.flow(train_targets.index, train_targets), epochs=5)
```

Comment:
Training and Evaluation section.

Reply:
I don't know what this means?

Reply:
Sorry the comments weren't linked at the right place. I meant was the the 

> Training and Evaluation

can start with the `compile` part. And the `model` line could be part of previous block. See the previous comment about that.

Reply:
Ah, cool. It's now like this.